recipes/quast/gnuplot-issue_disable-mummer-plots.patch

```diff
@@ -0,0 +1,26 @@
+From 02231ef13c093017f329abd6527da1474b5e670a Mon Sep 17 00:00:00 2001
```

Comment:
I think you renamed the wrong file :wink:

Reply:
yep, thanks! One more try to go :)